compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -676,6 +676,19 @@ OMR::SymbolReferenceTable::findOrCreateCurrentTimeMaxPrecisionSymbol()
    return element(currentTimeMaxPrecisionSymbol);
    }
 
+TR::SymbolReference *
+OMR::SymbolReferenceTable::findOrCreateEncodeASCIISymbol()
+   {
+   if (!element(encodeASCIISymbol))
+      {
+      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
+      sym->setHelper();
+
+      element(encodeASCIISymbol) = new (trHeapMemory()) TR::SymbolReference(self(), encodeASCIISymbol, sym);
+      }
+   return element(encodeASCIISymbol);
```

Comment:
It would probably be good to use `findOrCreateCodeGenInlinedHelper()` for this. See `J9::SymbolReferenceTable::findOrCreateNonNullableArrayNullStoreCheckSymbolRef()` for an example